software/spmd/expf/main.c

```diff
@@ -0,0 +1,17 @@
+#include <math.h>
+#include <stdio.h>
+
+float data = 3.4;
+
+int main() {
+  union {
+    int i;
+    float f;
+  } res;
+
+  res.f = expf(data);
+
+  printf("%x\n", res.i);
```

Comment:
the test should be self-checking. maybe add check the value and send the fail packet if it does not match the value expected by x86?